AAI/AAIConnectProfile.md

```diff
@@ -118,8 +120,8 @@ Access can be granted by either issuing new access tokens for downstream
 services (i.e. the Claim Clearinghouse may act like an authorization server)
 or by providing access to the underlying resources directly (i.e. the Claim
 Clearinghouse may act like a resource server). Some Claim Clearinghouses may
-issue access tokens that may contain a new set of GA4GH Claims and/or a
-subset of GA4GH claims for downstream consumption.
+issue tokens that may contain a new set of GA4GH Claims and/or a
+subset of GA4GH claims for downstream consumption (such as a Passport).
```

Comment:
"Passport" is an abstract concept, and existing specs and docs define it as: a logical set of identity from the access token returned from /authorize (implicit) or /token combined with the visa permissions returned from the /userinfo endpoint on such tokens.

This definition will be too narrow for what we want to do next, but is loosely worded enough in the spec itself to allow such changes while still being backwards compatible. However, for this sentence that talks about "tokens", and Passport is defined as "a GA4GH-compatible access token ... <plus other stuff>" and should likely not change that on a patch release.

I think you are introducing "token container" as part of a patch, and likely this should be a separate PR for v1.2 (given v1.1 should be skipped to avoid confusion with NIH).